tamr_client/transformations.py

```diff
@@ -0,0 +1,150 @@
+import requests
+from simplejson.errors import JSONDecodeError
+
+from tamr_client import dataset, response
+from tamr_client._types import Instance, JsonDict, Project, Session
+from tamr_client._types.transformations import InputTransformation, Transformations
```

Comment:
You should access all types from the `_types` package via `from tamr_client._types import <type>`. Looks like you correctly setup the `tamr_client/_types/__init__.py` to do this, so you should just be able to combine the import statements here.